src/Microsoft.ML.FastTree/FastTreeArguments.cs

```diff
@@ -174,16 +174,16 @@ public abstract class TreeOptions : LearnerInputBaseWithGroupId
         /// <summary>
         /// The seed of the random number generator.
         /// </summary>
-        [Argument(ArgumentType.LastOccurenceWins, HelpText = "The seed of the random number generator", ShortName = "r1")]
-        public int RngSeed = 123;
+        [Argument(ArgumentType.LastOccurenceWins, HelpText = "The seed of the random number generator", ShortName = "r1,RngSeed")]
+        public int RandomSeed = 123;
```

Comment:
>RandomSeed [](start = 19, length = 10)

I think everywhere else we call it just `seed`.

Reply:
Sure.

---
In reply to: [260562756](https://github.com/dotnet/machinelearning/pull/2753#discussion_r260562756) [](ancestors = 260562756)